projects/core/src/lib/users/quota/quota-info/quota-info.component.html

```diff
@@ -0,0 +1,23 @@
+<ng-container *ngIf="hasQuota()">
+    <span class="pair">
+        <span class="label">Available credits</span>
+        <span class="value">{{ sessionQuota?.available }} of {{ sessionQuota?.total }}</span>
+    </span>
+    <span class="pair">
+        <span class="label">Used credits (total)</span>
+        <span class="value">{{ sessionQuota?.used }} of {{ sessionQuota?.total }} </span>
+    </span>
+    <span class="pair">
+        <span class="label">Used credits (percent)</span>
+        <span class="value">{{ sessionQuota?.percentUsed }}%</span>
```

Comment:
Maybe you can use the percentage pipe from Angular?

https://angular.io/api/common/PercentPipe

Reply:
[9cbccf5](https://github.com/geo-engine/geoengine-ui/pull/586/commits/9cbccf56c567b103e556ba219e6260ed50dd9b63)